admin/new.docusaurus.io/functionUtils/playgroundUtils.ts

```diff
@@ -13,8 +13,7 @@ const PlaygroundConfigs = {
   // Do not use this one, see
   // https://github.com/codesandbox/codesandbox-client/issues/5683#issuecomment-1023252459
   // codesandbox: 'https://codesandbox.io/s/docusaurus',
-  codesandbox:
-    'https://codesandbox.io/s/github/facebook/docusaurus/tree/main/examples/classic',
+  codesandbox: 'https://codesandbox.io/p/sandbox/gallant-lamarr-4t1m4y',
```

Comment:
Is this expected to not have a clean URL?

Will it self-update if we update our `examples/classic` folder? 
(we count on this to make sure the playground is always on the latest version)

Reply:
I just updated the URL to use another sandbox, which will be synced with the repository and with a clear slug.